source/adapters/CMakeLists.txt

```diff
@@ -36,6 +36,13 @@ if(UR_BUILD_ADAPTER_L0 OR UR_BUILD_ADAPTER_CUDA OR UR_BUILD_ADAPTER_HIP)
     FetchSource(https://github.com/intel/llvm.git nightly-2023-08-01 "sycl/plugins/unified_runtime/ur" ${SYCL_ADAPTER_DIR})
 endif()
 
+# Temporarily fetch the opencl adapter from a fork until the PR has been merged.
+if(UR_BUILD_ADAPTER_OPENCL)
+    # fetch adapter sources from SYCL
+    set(SYCL_ADAPTER_DIR "${CMAKE_CURRENT_SOURCE_DIR}/external")
+    FetchSource(https://github.com/fabiomestre/llvm.git opencl_adapter_unofficial "sycl/plugins/unified_runtime/ur" ${SYCL_ADAPTER_DIR})
```

Comment:
perhaps this could be done within `adapters/opencl/CMakeLists.txt`?

Reply:
Since all the other adapters are doing the same operation in this CMakeLists file, I think it makes sense to keep it here. Once the adapter is merged, this will be deleted and added to the if case above.

Reply:
👍 
mea culpa, I missed that somehow